tests/plugins/subcommands/doctor/test_health_checks.py

```diff
@@ -242,6 +244,56 @@ def test_not_env_txt_check_action(
     assert X_MARK in captured.out
 
 
+def test_requests_ca_bundle_check_action_passes(
+    env_ok: tuple[Path, str, str, str, str],
+    capsys,
+    monkeypatch,
```

Comment:
Let's use typing for all of these:

```suggestion
    capsys: CaptureFixture,
    monkeypatch: MonkeyPatch,
```